### PR TITLE
[MessageBus] Fixed public service/alias compiler pass

### PR DESCRIPTION
**Pull request by brusch**
Follow-up to #10329

### CODE REVIEW

File: bundles/CoreBundle/DependencyInjection/Compiler/MessageBusPublicPass.php [hunk 1]
```diff
@@ -28,6 +28,6 @@ final class MessageBusPublicPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container)
     {
-        $container->findDefinition(MessageBusInterface::class)->setPublic(true);
+        $container->getAlias(MessageBusInterface::class)->setPublic(true);
```
Comment (dpfaffenbauer) [inline bundles/CoreBundle/DependencyInjection/Compiler/MessageBusPublicPass.php:31]: use `find` instead, checks for definition or alias

Comment (brusch) [reply bundles/CoreBundle/DependencyInjection/Compiler/MessageBusPublicPass.php:31]: True, will check that, thanks!

Comment (brusch) [reply bundles/CoreBundle/DependencyInjection/Compiler/MessageBusPublicPass.php:31]: `findDefinition()` is not an option, it returns the service, but `setPublic()` on the alias doesn't work (only on the target service, which is `messenger.default_bus`).

Comment (dpfaffenbauer) [reply bundles/CoreBundle/DependencyInjection/Compiler/MessageBusPublicPass.php:31]: alright

**Pull request merged**